e2e/specs/st_spinner.spec.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ * Copyright 2018-2021 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("st.spinner", () => {
+  before(() => {
+    cy.visit("http://localhost:3000/");
+  });
+
+  it("displays a message", () => {
+    cy.get(".stButton button").click(); // click button to trigger spinner execution
+    cy.get(".element-container .stSpinner").should("contain", "Loading...");
+  });
+
+  it("displays a spinner svg", () => {
+    cy.get(".stButton button").click(); // click button to trigger spinner execution
```

Comment:
I think `trigger spinner execution` might be a comment but that's just me.

Reply:
I added the comment `click button to trigger spinner execution` since it might not be super obvious why a button click is needed in this situation. Is there a better way to do that?

Reply:
Oops I did not properly type out my comment. I meant `trigger spinner execution` might be a *better* comment because line 29 already has click and button in it instead of `click button to trigger spinner execution`.